examples/github-appcards/src/components/GitHub/GitHub.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import importGithub from "../../assets/import-github.png";
+
+const GitHub = () => {
+  const handleChooseGithubClick = async () => {
+    miro.board.ui.openModal({
+      url: "modal.html",
+      fullscreen: false,
+    });
+  };
+
+  return (
+    <div id={"github-container"}>
+      <h3>Synced changes</h3>
+      <p>
+        Any changes you make in Miro or GitHub will be synced between both
+        tools.
```

Comment:
Suggestion:

"Any changes you apply, either in Miro or in GitHub, are synced between both tools."